modules/virt-installing-a-watchdog.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog_{context}"]
+= Installing a watchdog device
+
+To activate a watchdog device, install the `watchdog` package on your virtual machine and start the watchdog service.
```

Comment:
I think you could remove "To activate ...." 
"Install ... and start ..." makes a good intro sentence.